### DISCUSSION TITLE
What are these battery icons?

### DISCUSSION THREAD

**Discussion by v4hn** [starship/starship#6051] [Q&A] [answered]
I've been using starship for a few years now and whenever my battery is low the default prompt shows this character:

<img src="https://github.com/starship/starship/assets/680358/fc91734e-702e-49f0-8344-0c52e945041b" width="100px">

I wanted to know why.

Copying the UTF-8 and searching didn't work for me, nor did image search, or asking multiple Chinese (mainland). Another friend (Taiwanese) found it in traditional Chinese as `tuo'`: https://www.zdic.net/hant/%F0%A5%B4%B0
It would have helped a lot if the meanings were explained in the source definition...

But now here's my question: Is this the intended character to be shown or an issue with my fonts? Why show something that means "guiding an [ox] cart" when the battery is low?

**Comment by v4hn** [comment, marked as answer]
Funny how I was happy with this for years and never looked into it...
Turns out I failed to fulfill the [only prerequisite for startship](https://starship.rs/#prerequisites) and did not have a nerdfont enabled.
With the font enabled I get more reasonable (though less-stylish ;)) icons like this:

![screenshot240623-212437](https://github.com/starship/starship/assets/680358/7a40f896-b675-489e-921d-c2b103d0014c)

**Reply by davidkna** [reply]
Recent nerd font versions have migrated not to use any codepoints that encroach on other characters outside the private-use areas. 

Starship has since been updated to only use these updated codepoints. If you use any older preset files, you might need to update them.

**Reply by v4hn** [reply]
Yes, I had to upgrade to a new starship version after installing a current nerdfont. That explains it, thanks.
